src/UI/components/map/layers/listButton.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import Checkbox from '@mui/material/Checkbox';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemText from '@mui/material/ListItemText';
+
+const ListButton= ({name}:{name:string}) => {
+  return (
+    <>

```

Comment:
I don't think you need this empty `<>` - these are used for when you have multiple things to render at the same level. You only have one `ListItemButton` here, so I think we should be okay.